gradle-examples/jaxrs-cdi/src/test/java/org/wildfly/swarm/examples/gradle/ArqWarDeploymentTest.java

```diff
@@ -37,12 +37,7 @@ public static Archive createDeployment() throws Exception {
         archive.addClass(HelloRest.class);
         archive.addClass(MyApplication.class);
         archive.addAsWebInfResource(EmptyAsset.INSTANCE, "beans.xml");
-
-        // Include the test case related dependencies (non-fractions) explicitly.
-        // Do not use the addAllDependencies() as that will pull in dependencies that won't be satisfied in a test case.
-        // If you ended up on https://issues.jboss.org/browse/THORN-1072 via a google search, then make sure that you remove
-        // the addAllDependencies() invocation.
-        archive.addDependency("commons-io:commons-io:2.4");
```

Comment:
With the recent set of changes to module packaging, we no longer need to worry about pulling in the wrong set of jars that cause startup failures. There are a few issues that I am observing with how _webapp_ projects retrieve dependencies, but I can handle that as part of the existing plugin setup in a separate PR.